.circleci/config.yml

```diff
@@ -21,6 +21,7 @@ workflows:
           requires:
             - test-node-v6
             - test-node-v8
+      - conda-build-and-upload
```

Comment:
We probably only want to run this if `test-node-v6` and `test-node-v8` succeed. Adding 

```
  requires:
     - test-node-v6
     - test-node-v8
```

should do the trick

Reply:
... and we should probably rename this step `conda-build` as we won't be uploading the builds on CI.